### PR TITLE
Update: Keep ratio on cover video backgrounds

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
This PR updates cover block to use object-fit cover instead of object-fit fill for the video background.
This change will make sure the ratio of the video is always kept.
The downside is that part of the video may be cropped to make sure it fills the background, and its ratio is kept.

## How has this been tested?
Verify the cover block still works as before.
Verify the ratio of the video is always kept (by resizing the window) in the frontend and it the editor.

## Screenshots
After:
nov-15-2018 00-02-22
Before:
nov-15-2018 00-07-48

### CODE REVIEW

File: packages/block-library/src/cover/style.scss [hunk 1]
```diff
@@ -10,6 +10,7 @@
 	display: flex;
 	justify-content: center;
 	align-items: center;
+	overflow: hidden;
```
Comment (mtias) [inline packages/block-library/src/cover/style.scss]: Why is this needed for both image and video?

Comment (jorgefilipecosta) [reply packages/block-library/src/cover/style.scss]: This is only needed for the videos. But it does not affect the images or changes anything if also applied to images. We don't have a way in CSS to target a class if it contains a specific child e.g: a video, so I applied to both images and video.

**Comment by karmatosed** [comment]
This to me makes sense. My only wonder is cross browser support but a cautious look seems 👍

**Comment by jasmussen** [comment]
This works for me in my testing. I can confirm that the aspect ratio before was skewed, and it isn't anymore. The extra crop is fine. 

I was concerned the added overflow rule would make the sticky toolbars not work — it has done so in the past. But the sticky toolbar works fine here, so the scope for the rule is good.

👍 👍

**Review by youknowriad — APPROVED** [review]
LGTM 👍
